### PR TITLE
update clean method to compeletely remove temp files

### PR DESCRIPTION
**Pull request by hainm**

### CODE REVIEW

**Comment by hainm** [comment]
so in AMBER, just need to (cd parmed && $(PYTHON) setup.py clean) to remove temp files.

**Comment by swails** [comment]
Looks reasonable to me.  Thanks!

File: setup.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import os
 import sys
 
+if sys.version_info < (2, 7):
+    sys.stderr.write('You must have at least Python 2.7 for ParmEd to work '
+                     'correctly.\n')
+    sys.exit(0)
```
Comment (swails) [inline setup.py]: This needs to be non-zero.

Comment (hainm) [inline setup.py]: updated.

**Pull request merged**